lib/ansible/config/base.yml

```diff
@@ -215,6 +215,14 @@ CACHE_PLUGIN_TIMEOUT:
   - {key: fact_caching_timeout, section: defaults}
   type: integer
   yaml: {key: facts.cache.timeout}
+COLLECTIONS_PATHS:
+  name: ordered list of root paths for loading installed Ansible collections content
+  default: ['~/.ansible/collections', '/usr/share/ansible/collections']
```

Comment:
default is not matching type, needs to be `'~/.ansible/collections:/usr/share/ansible/collections'`

Reply:
pathspec allows both lists and strings; why use a compound datatype as the default?

Reply:
cause it shows in docs and people get confused in `ini` and `env vars` when writing lists

Reply:
done